bridge_common.spec

```diff
@@ -0,0 +1,58 @@
+Name: tendrl-bridge-common
+Version: 0.0.1
+Release: 1%{?dist}
+BuildArch: noarch
+Summary: Common Module for all Bridges
+Source0: %{name}-%{version}.tar.gz
+Group:   Applications/System
+License: LGPLv2+
+URL: https://github.com/Tendrl/bridge_common
+
+BuildRequires: systemd
+BuildRequires: python2-devel
+BuildRequires: pytest
+
+Requires: python-etcd
+Requires: python-dateutil
+Requires: python-hacking
+Requires: pytest
+Requires: python-pytest-cov
```

Comment:
pytest, python-hacking, python-pytest-cov, and python-mock, python-reno, and python-coveralls all look like `BuildRequires`, not `Requires`?

Did you try building this package in mock for EPEL 7?